src/window.rs

```diff
@@ -874,3 +891,23 @@ pub enum Theme {
     Light,
     Dark,
 }
+
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum RequestUserAttentionType {
+    /// ## Platform-specific
+    /// - **macOS:** Bounces the dock icon until the application is in focus.
+    /// - **Windows:** Flashes both the window and the taskbar button until the application is in focus.
+    /// - **X11:** Sets the WM's `XUrgencyHint`. No distinction between `Critical` and `Informational`.
+    Critical,
+    /// ## Platform-specific
+    /// - **macOS:** Bounces the dock icon once.
+    /// - **Windows:** Flashes the taskbar button until the application is in focus.
+    /// - **X11:** Sets the WM's `XUrgencyHint`. No distinction between `Critical` and `Informational`.
```

Comment:
That's a lot of text I have to read just to find out that most of these are no different on X11. Might be useful to add a doc comment on the enum itself and describe that there?

Reply:
Agreed! Moved it to the enum itself instead of the variants.